shuffle.py

```diff
@@ -581,19 +569,17 @@ def populate(self):
         for (dirpath, dirnames, filenames) in os.walk(self.path):
             dirnames.sort()
             # Ignore the speakable directory and any hidden directories
-            if "ipod_control/speakable" not in dirpath.lower() and "/." not in dirpath.lower():
+            if "iPod_Control/Speakable" not in dirpath and "/." not in dirpath:
```

Comment:
I am a little blurred on whether to include hidden directories or not. The original intention here could have been that one doesn't store user data within hidden directories, or allowing other apps to store their own stuff inside these oblivious to the user.